test/unit/format_capture_helper_test.rb

```diff
@@ -0,0 +1,87 @@
+require_relative "../test_helper"
+
+module MockCaptureHelper
+  def content_for(_name, content, _options, &_block)
+    content
+  end
+
+  def capture(&block)
+    block.call
+  end
+
+  def render(*_args, &block)
+    "<g>" + block.call + "</g>"
```

Comment:
It has taken me a bit of a while to work out how this works - I wonder if it can be made clearer. As I understand we can assume that this is govspeak because that is the only time we call render?

It might be a bit easier to follow if it was along the lines of:

```
  def render(component, &block)
    if component == "govuk_publishing_components/components/govspeak"
      "<govspeak>#{block.call}</govspeak>"
    else
       block.call
    end
  end
```

Reply:
Yup - added this in.